presidio-structured/presidio_structured/analysis_builder.py

```diff
@@ -154,18 +154,26 @@ class TabularAnalysisBuilder(AnalysisBuilder):
 class PandasAnalysisBuilder(TabularAnalysisBuilder):
     """Concrete configuration generator for tabular data."""
 
+    entity_selection_strategies = {"highest_confidence", "mixed", "most_common"}
+
     def generate_analysis(
```

Comment:
Overall, PR looks good to me, nice addition!

One thing which might be nice to address (although you didn't introduce this issue) is that we are redefining the function definition of generate_analysis from the AnalysisBuilder(ABC)

Perhaps we can address it in this PR as well, and redefine the abstract method in the base class to: 

`
    @abstractmethod
    def generate_analysis(
        self,
        data: Union[Dict, DataFrame],
        *args,
        **kwargs
    ) -> StructuredAnalysis:
`

(.. and then rename df to data to be consistent)

So we don't override the function definition in the concrete implementations of the class.

Reply:
I suggest to handle this in a subsequent PR if that's ok with everyone.